Remove Anomalyze and update READMEs (#67)
WayScience/JUMP-single-cell: 3.analyze_data/README.md

# Analyze Predicted Probabilities

> In this module, we perform analyses on predicted probability data to evaluate phenotypic behavior across treatments (for example, compound, CRISPR, and ORF).

Reviewer: ```suggestion
In this module, we perform analyses on predicted probability data to evaluate phenotypic behavior across treatments per dataset (e.g., compound, CRISPR, and ORF).
```
Wouldn't this be more accurate?

Author: What do you mean by `per dataset` here? Aren't we only using the jump pilot dataset (cpg0000)?

Reviewer: This was an assumption, I believe that compound, CRISPR and ORF are individual "datasets" or treated as such. Feel free to ignore or change the wording!

Core comparisons use KS-based testing against negative control groups.

## Analyze Well Probabilities
We compare the phenotype probabilities between each treated well and the remaining negative control wells on the corresponding plate.
Each treatment well and corresponding negative control well phenotype probabilities are only compared if the number of cells in these groups is above a given cell count threshold.
The group, treatment cells or control cells, are then randomly down-sampled depending on which of these groups has a larger population of cells.

> Random sampling of control cells is done through stratification by plate well.

Reviewer: ```suggestion
Random sampling of control cells is done through stratification by plate and well.
```

After sampling the cell population, the cells from the treated and control groups are compared using the KS test statistic.

> The analysis scripts in this directory include plate-level and treatment-level aggregation, significance testing, and probability distribution visualization.

Reviewer: Do you perform another round of aggregation? This is a bit confusing. I am not familiar with what plate-level aggregation would be. Would it mean all cells and wells per plate are averaged? Recommend clarifying.

Author: Sounds good, I will clarify this
